jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/WebAppGeneratorTest.java

```diff
@@ -169,4 +171,47 @@ public void customizeWithOverriddenPropertiesShouldAddImageConfiguration() throw
         .extracting("inline.files").asList().extracting("destName")
         .containsExactly("some-context.war");
   }
+
+  @Test
+  public void extractEnvVariables_withMultipleEnvVariableSpaceSeparator_shouldExtractThemAll() {
+    // Given
+    String envConfig = "GALLEON_PROVISION_LAYERS=web-server,ejb-lite,jsf,jpa,h2-driver IMAGE_STREAM_NAMESPACE=myproject";
+    // when
+    Map<String, String> extractedVariables = WebAppGenerator.extractEnvVariables(envConfig);
+    // then
+    assertEquals("web-server,ejb-lite,jsf,jpa,h2-driver", extractedVariables.get("GALLEON_PROVISION_LAYERS"));
+    assertEquals("myproject", extractedVariables.get("IMAGE_STREAM_NAMESPACE"));
+  }
+
+  @Test
+  public void extractEnvVariables_withSingleEnvVariable_shouldExtractIt() {
+    // Given
+    String envConfig = "GALLEON_PROVISION_LAYERS=web-server,ejb-lite,jsf,jpa,h2-driver";
+    // when
+    Map<String, String> extractedVariables = WebAppGenerator.extractEnvVariables(envConfig);
+    // then
+    assertEquals("web-server,ejb-lite,jsf,jpa,h2-driver", extractedVariables.get("GALLEON_PROVISION_LAYERS"));
+  }
+
+  @Test
+  public void extractEnvVariables_withMultipleEnvVariableCRSeparator_shouldExtractThemAll() {
+    // Given
+    String envConfig = "GALLEON_PROVISION_LAYERS=web-server,ejb-lite,jsf,jpa,h2-driver\nIMAGE_STREAM_NAMESPACE=myproject";
```

Comment:
This contradicts with: https://github.com/eclipse/jkube/pull/1351/files#r831139053

However, I'm not sure if it's likely for someone to provide an env variable with an LF